scanner/updater/export.go

```diff
@@ -20,15 +20,15 @@ import (
 )
 
 // Export is responsible for triggering the updaters to download Common Vulnerabilities and Exposures (CVEs) data
-// and then outputting the result as a zstd-compressed file with .ztd extension
+// and then outputting the result as a zstd-compressed file with .zst extension
 func Export(ctx context.Context, outputDir string) error {
 
 	err := os.MkdirAll(outputDir, 0700)
 	if err != nil {
 		return err
 	}
 	// create output json file
-	outputFile, err := os.Create(filepath.Join(outputDir, "output.json.ztd"))
+	outputFile, err := os.Create(filepath.Join(outputDir, "output.zst"))
```

Comment:
why remove the `.json`?

Reply:
also just noticed we don't close the file. That may help with syncing, too

Reply:
> why remove the `.json`?

I added the 'json' back and defer close the output file.